.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,46 +1,34 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: "[BUG] "
-labels: bug
-assignees: Konijima
+## Project Zomboid Version
+[e.g. 41.56-IWBUMS]
 
----
-
-**Project Zomboid Version**
-Your current game version.
-
-**Describe the bug**
+## Describe the bug  
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## To Reproduce 
 Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
+## Expected behavior
 A clear and concise description of what you expected to happen.
 
-**Screenshots**
+## Screenshots
 If applicable, add screenshots to help explain your problem.
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+## Operating System
+[e.g. Win10 64 bit, linux ubuntu 21.10]
 
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
-
-**Enabled Mods**
+## List of enabled Mods when the bug occured
 - Mod 1
 - Mod 2
 - Mod 3
 
-**Additional context**
-Add any other context about the problem here.
+## Your latest console.txt when the bug occured
+On Windows found at : `C:\Users\<Username>\Zomboid\console.txt`
+
+To quickly go to that location, you can copy `%UserProfile%\Zomboid\` into the address bar on the top of the file explorer.
+```
```

Comment:
It should be noted this log resets when the game is started up, so the log has to be retrieved after the error occurs but before the game is restarted.